examples/sgp4Prop/fsw/src/TestSGP4.c

```diff
@@ -138,7 +138,7 @@ void runVER(VERIN *verins, int cnt)
         fclose(in_file);
     }
 }
-/*
```

Comment:
Can you explain why you made this change (uncommenting this function)? I'm pretty sure this was commented out because we don't want this to be the entry point into the code... we probably wanted to be able to compile a function from this file into our code without making it a standalone executable (since we want it to be a CFS app), but I can't remember.

Reply:
I uncommented this function a while back when I was trying to get the app to run initially, was doing some debugging, I ended up re-commenting it.